README.md

```diff
@@ -251,6 +251,14 @@ when initializing your instance of the SplunkRum API:
 - `diskBufferingEnabled(boolean)` : 
   Enables the storage-based buffering of telemetry. 
   This setting is useful when instrumenting applications that might work offline for extended periods of time.
+- `sessionBasedSamplingEnabled(boolean)` :
+  Enables session ID based sampling for traces. Ratio must be set with `sessionBasedSamplingRatio`,
+  as the default ratio includes all sessions. Sampling is only applied if the spans created via 
+  `SplunkRum` APIs are root spans. If a custom parent span context was created, then the sampling 
+  flag of the parent span is respected instead.
+- `sessionBasedSamplingRatio(double)` :
+  Set sampling ratio for session ID based sampling. This is a probability of a session being
+  included between between 0.0 (all dropped) and 1.0 (all included). Default is 1.
```

Comment:
in this case, since the default is to not sample (i.e. a value of 1.0), that we don't need 2 settings for this. Just one method like `enableSessionBasedSampling(double)` seems like it should be clear and concise enough to cover all the bases.

Reply:
Done. Kept the boolean field internally as comparing that during initialization seems to be cleaner than comparing a double to `1.0`.